src/tests/ftest/launch.py

```diff
@@ -953,6 +958,161 @@ def rename_logs(avocado_logs_dir, test_file):
+            #         'release': pkg_data['release']}
+            #yum_base.install(**kwarg)
+            install_pkgs.append({'name': debug_pkg,
+                                 'version': pkg_data['version'],
+                                 'release': pkg_data['release'],
+                                 'epoch': pkg_data['epoch']})
+
+    # This is how you normally finish up a yum transaction, but
+    # again, we need to employ sudo
+    #yum_base.resolveDeps()
+    #yum_base.buildTransaction()
+    #yum_base.processTransaction(rpmDisplay=yum.rpmtrans.NoOutputCallBack())
+    cmd = "sudo yum -y --enablerepo=\\*debug\\* install"
+    for pkg in install_pkgs:
+        try:
+            cmd += " {}-{}-{}".format(pkg['name'], pkg['version'],
+                                      pkg['release'])
+        except KeyError:
+            cmd += " {}".format(pkg['name'])
+    cmds.append(cmd)
+
+    print(get_output(';'.join(cmds)))
+
+
+def process_the_cores(avocado_logs_dir, test_yaml, args):
+    """Copy all of the host test log files to the avocado results directory.
+
+    Args:
+        avocado_logs_dir ([type]): [description]
+        test_yaml (str): yaml file containing host names
+        args (argparse.Namespace): command line arguments for this program
+    """
+    import fnmatch
+
+    this_host = socket.gethostname().split(".")[0]
+    host_list = get_hosts_from_yaml(test_yaml, args)
+    daos_cores_dir = os.path.join(avocado_logs_dir, "latest", "stacktraces")
+
+    # Create a subdirectory in the avocado logs directory for this test
+    print("Processing cores from {} in {}".format(host_list, daos_cores_dir))
+    get_output("mkdir {}".format(daos_cores_dir))
+
+    # Copy any core files that exist on the test hosts and remove them from the
+    # test host if the copy is successful.  Attempt all of the commands and
+    # report status at the end of the loop.  Include a listing of the file
+    # related to any failed command.
+    commands = [
+        "set -eu",
+        "rc=0",
+        "copied=()",
+        "for file in /var/tmp/core.*",
+        "do if [ -e $file ]",
+        "then if scp $file {}:{}/${{file##*/}}-$(hostname -s)".format(
+            this_host, daos_cores_dir),
+        "then copied+=($file)",
+        "if ! sudo rm -fr $file",
+        "then ((rc++))",
+        "ls -al $file",
+        "fi",
+        "else ((rc++))",
+        "ls -al $file",
+        "fi",
+        "fi",
+        "done",
+        "echo Copied ${copied[@]:-no files}",
+        "exit $rc",
+    ]
+    spawn_commands(host_list, "; ".join(commands), timeout=1800)
+
+    cores = os.listdir(daos_cores_dir)
+
+    if not cores:
+        return
+
+    install_debuginfos()
+
+    def run_gdb(pattern):
+        """Run a gdb command on all corefiles matching a pattern.
+
+        Args:
+            pattern (str): the fnmatch/glob pattern of core files to
+                           run gdb on
+        """
+        import magic
```

Comment:
(pylint-import-error) Unable to import 'magic'

Reply:
I don't know how to resolve this.  I have added the RPM containing the module to the system before `pylint` is run.

Reply:
Same as above ... Typically we need to install the python3 version of the magic module for it to be detected by the python3 version of pylint that we now run.

Reply:
Ah right.  We really need to get off of this dual python version bandwagon.  But then again, that would actually cripple us due the lagginess of python3 on EL7.

In any case, there is no python3 version of this module on EL7.